README.md

```diff
@@ -79,11 +79,17 @@ Clone this repository to your desired folder: (https://github.com/HERMON-1995/Le
   git clone [https://github.com/HERMON-1995/Leaderboard-list-app-project.git]
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/223365364-c77bee71-4667-47b5-ad06-0e9b13bddbd0.png)

**[Optional]**

- [ ] You have two Key Feature sections:thinking: Kindly remove the first one because that one is not professional and there should be only one Key Feature section :blush:
- [ ] It would be nice if you put some sensible features such as those shown below to make your readme file descriptive :+1:

* Add Score
* Sort Score